Remotion/Data/DomainObjects.Web.Test/Performance/Form.aspx

```diff
@@ -140,7 +140,7 @@
     prm.add_pageLoading(Page_OnBeginAsyncLoading);
     prm.add_endRequest(Page_OnEndAsyncRequest);
 
-    $(window).load(Page_OnEndSyncRequest);
+    $(window).load(Page_OnEndSyncRequest); // TODO: what is this? .load(string) is the actually signature
```

Comment:
This executes the Page_OnEndSyncRequest function when the window.load event is fired. 

https://api.jquery.com/load-event/#load-handler
jQuery doc says `load (function(){...})` works, so passing an existing function by handle also works.